source/crud/tests/write/bulkWrite-arrayFilters.yml

```diff
@@ -0,0 +1,42 @@
+data:
+    - {_id: 1, y: [{b: 3}, {b: 1}]}
+    - {_id: 2, y: [{b: 0}, {b: 1}]}
+
+minServerVersion: '3.5.6'
+
+tests:
+    -
+        description: "BulkWrite with arrayFilters"
+        operation:
+            name: "bulkWrite"
+            arguments:
+                requests:
+                    -
+                        name: "updateOne"
+                        arguments:
+                            filter: {}
+                            update:
+                                $set: {"y.$[i].b": 2}
+                            arrayFilters:
+                              - {i.b: {"$eq": 3}}
```

Comment:
This looks to be modeled off of "UpdateOne when one document matches arrayFilters" in `updateOne-arrayFilters.yml`. 
Why `{$eq: 3}` instead of `{i.b: 3}`?